src/app/src/SensorMarker.js

```diff
@@ -34,16 +25,21 @@ export default function SensorMarker(props) {
     };
 
     return (
-        <svg
-            height={size}
-            viewBox='0 0 24 24'
-            style={{
-                ...pinStyle,
-                transform: `translate(${-size / 2}px,${-size}px)`,
-            }}
-            onClick={handleOnClick}
-        >
-            <path d={ICON} />
-        </svg>
+        <div onClick={handleOnClick}>
+            <div className='marker marker--positive'>
+                <div className='marker__inner'>
+                    <div className='marker__content'>
+                        <div className='marker__level' />
+                        <div className='marker__symbol'>
+                            <img
```

Comment:
What is the purpose for all these nested divs? Are these layered classes how we get the bouncing fish animation?

Reply:
I believe so! I copied this mark-up from the prototype, which @alexelash designed. She may be able to provide more insight.